packages/firestore/src/util/obj_map.ts

```diff
@@ -17,7 +17,7 @@
 import { Equatable } from './misc';
 import * as objUtil from './obj';
 
-type Entry<K, V> = [K, V];
+export type Entry<K, V> = [K, V];
```

Comment:
We already export a type Entry in sorted_map.ts. I would actually prefer if we use that type since it's easier to digest (it's an object on not an array)

Reply:
If we do keep it, then please rename it. Having two Entry types that are very similar will make auto-importing a bit tricky.